pkg/cmd/tasks/execute/execute.go

```diff
@@ -238,3 +244,17 @@ func slugFromScript(file string) (string, error) {
 
 	return slug, nil
 }
+
+type notDeployedError struct {
+	task string
+}
+
+// Error implementation.
+func (err notDeployedError) Error() string {
+	return fmt.Sprintf("task %s was not deployed", err.task)
```

Comment:
minor wording tweak + I think error messages user see should be capitalized

```suggestion
	return fmt.Sprintf("Task %s has not been deployed", err.task)
```

Reply:
Let's capitalize when we render instead.